pkg/occlient/occlient.go

```diff
@@ -96,7 +96,8 @@ const (
 
 	// Default Image that will be used containing the supervisord binary and assembly scripts
 	// use getBoostrapperImage() function instead of this variable
-	defaultBootstrapperImage = "registry.access.redhat.com/openshiftdo/odo-init-image-rhel7:1.0.0"
+	// defaultBootstrapperImage = "registry.access.redhat.com/openshiftdo/odo-init-image-rhel7:1.0.0"
+	defaultBootstrapperImage = "yusufk53/odo-init-image:1066"
```

Comment:
Cool!!!
Do not forget to remove this after a successful CI run

Reply:
Sure!